nuxt.config.js

```diff
@@ -78,4 +78,7 @@ export default {
     }
   },
   ignore: ['**/*.test.*', '**/*.spec.*'],
+  publicRuntimeConfig: {
+    baseURL: process.env.BASE_URL || 'http://localhost:8080/api',
```

Comment:
where will we use this variable?

Reply:
It is baseUrl for api server. But I configured this URL in axios option. So I will delete it.

Reply:
Ok 👍